addons/csw/CfgMagazines.hpp

```diff
@@ -0,0 +1,89 @@
+class CfgMagazines {
+    class 100Rnd_127x99_mag;
+    class 1Rnd_GAT_missiles;
+    class 40Rnd_20mm_G_belt;
+    class 1Rnd_GAA_missiles;
+    
+    class GVAR(100Rnd_127x99_mag): 100Rnd_127x99_mag {
+        author = ECSTRING(common, ACETeam);
+        displayName = CSTRING(HMGBelt_displayName);
+        model = "\A3\Structures_F_EPB\Items\Military\Ammobox_rounds_F.p3d";
+        scope=2;
```

Comment:
Could you adjust the spacing so there is a space infront and behind each `=`? 

For example on line 11, change it to:
```
scope = 2;
```

This will keep everything uniform and clean. This occures on multiple locations in this file.